packages/framework-core/src/services/event-store.ts

```diff
@@ -42,20 +40,43 @@ export class EventStore {
         newEntitySnapshot
       )
 
-      if (newEntitySnapshot && pendingEvents.length >= numberOfEventsBetweenSnapshots) {
-        await this.storeSnapshot(newEntitySnapshot)
-      }
-
```

Comment:
This function is now safe to be used by any handler, as it is not generating snapshots anymore